pytest_fixtures/core/sat_cap_factory.py

```diff
@@ -95,8 +169,10 @@ def module_capsule_configured(module_capsule_host, module_target_sat):
 @pytest.fixture(scope='session')
 def session_capsule_configured(session_capsule_host, session_target_sat):
     """Configure the capsule instance with the satellite from settings.server.hostname"""
-    session_capsule_host.capsule_setup(sat_host=session_target_sat)
-    yield session_capsule_host
+    if session_capsule_host:
+        session_capsule_host.capsule_setup(sat_host=session_target_sat)
+        yield session_capsule_host
+    yield
```

Comment:
```suggestion
    if session_capsule_host:
        session_capsule_host.capsule_setup(sat_host=session_target_sat)
        yield session_capsule_host
```

Reply:
@jyejare Can you check this asap as puppet capsule tests are still failing because of this?

Reply:
```pytest_fixtures/component/puppet.py:18: in session_puppet_enabled_capsule
    yield session_capsule_host.enable_puppet_capsule(satellite=session_puppet_enabled_sat)
robottelo/host_helpers/capsule_mixins.py:22: in enable_puppet_capsule
    assert result.status == 0
E   AssertionError```

Reply:
I dont see its related to this change , do we have PR to fix that issue in Master so that I can pull the changes in this PR.

Reply:
@jyejare I'm referring to this PR change https://github.com/SatelliteQE/robottelo/pull/11977#issuecomment-1649819453

Reply:
Ahh Done !